jerry-core/ecma/builtin-objects/ecma-builtin-json.c

```diff
@@ -1400,6 +1432,21 @@ ecma_builtin_json_serialize_property (ecma_json_stringify_context_t *context_p,
     return ECMA_VALUE_EMPTY;
   }
 
+#if ENABLED (JERRY_ESNEXT)
+  if (ecma_is_value_bigint (value))
+  {
+    if (value != ECMA_BIGINT_ZERO)
+    {
+      ecma_deref_bigint (ecma_get_extended_primitive_from_value (value));
+    }
+    else
+    {
+      ecma_free_value (value);
+    }
+    return ecma_raise_type_error (ECMA_ERR_MSG ("value is BigInt"));
```

Comment:
This error message could be a bit more descriptive.

Reply:
Yes, something wihch tells we are in json serialize